### DISCUSSION TITLE
Bind mount failed: '/volume1/@docker/import' does not exists

### DISCUSSION THREAD

**Discussion by GitHub-erNik** [teslamate-org/teslamate#2665] [Q&A] [answered]
Hi, I was trying to reinstall my TeslaMate since it was no longer accessible by :4000. I successfully created a a backup. So I uninstalled everything within docker and docker itself. Then I was running the .yml file again. This time it failed (tried like 3x times in a row, always cleaning up everything afterwards). I get this Error:

```
ERROR: for docker_teslamate_1  Cannot start service teslamate: Bind mount failed: '/volume1/@docker/import' does not exists

ERROR: for teslamate  Cannot start service teslamate: Bind mount failed: '/volume1/@docker/import' does not exists
```

My file is 1:1 this https://docs.teslamate.org/docs/installation/docker (except the passwords of course)

![image](https://user-images.githubusercontent.com/106312055/170471827-f6f58572-90c6-47e1-b2d2-28a01ecb4e25.png)

Any help is highly appricated!

**Comment by GitHub-erNik** [comment, marked as answer]
*was able to fix it by just manually mkdir the missing folder before running the .yml file

**Comment by GPSBabelDeveloper** [comment]
That's the correct answer, but for those shaky with their shell commands, the thing to type is:

 mkdir /volume1/@docker/import
 
 (where mkdir is the command to make the directory and /volume.... is the literal text from the error message above telling you what's missing, that was specified in the 'import' field of the yml.)
 
 Thanx and enjoy!
